fakerator.d.ts

```diff
@@ -104,7 +103,7 @@ declare module 'fakerator' {
          * @param precision Can generate floating number if you define this parameter. Note: Do not use 0
          * @returns number
          */
-        number(min: number, max: number, precision: number = 1): number;
+        number(min: number, max: number, precision: number): number;
```

Comment:
> Why do you remove the default value of method parameters?

They are not a valid TS construct. Default values are only for runtime declarations, not type declarations. Actually I just realized the function parameters with default values should be optional

```suggestion
        number(min: number, max: number, precision?: number): number;
```